gabbi/runner.py

```diff
@@ -88,27 +88,59 @@ def run():
         help='Custom response handler. Should be an import path of the '
              'form package.module or package.module:class.'
     )
+    parser.add_argument(
+        '-f',
+        nargs='?', default=None,
+        dest='input_files',
+        action='append',
+        help='input files'
+    )
 
     args = parser.parse_args()
     host, port, prefix, force_ssl = utils.host_info_from_target(
         args.target, args.prefix)
 
-    # Initialize response handlers.
-    handler_objects = initialize_handlers(args.response_handlers)
+    response_handlers = initialize_handlers(args.response_handlers)
```

Comment:
This variable rename seems extraneous. Though the argument remains response_handlers to preserve backward compatibility and not add noise to the system, we're in a world of content-handlers _and_ response-handlers now, thus handler_objects.

However, it doesn't really matter, just noting that I noticed.